api/src/pcapi/core/educational/repository.py

```diff
@@ -720,34 +691,39 @@ def find_collective_bookings_by_pro_user(
     for booking in collective_bookings_page:
-        booking_with_converted_dates = CollectiveBookingNamedTuple(
-            collectiveBookingId=booking.collectiveBookingId,
-            bookedAt=convert_booking_dates_utc_to_venue_timezone(booking.bookedAt, booking),
-            bookingAmount=booking.bookingAmount,
-            usedAt=convert_booking_dates_utc_to_venue_timezone(booking.usedAt, booking),
-            cancelledAt=convert_booking_dates_utc_to_venue_timezone(booking.cancelledAt, booking),
-            cancellationLimitDate=convert_booking_dates_utc_to_venue_timezone(booking.cancellationLimitDate, booking),
-            confirmationLimitDate=convert_booking_dates_utc_to_venue_timezone(booking.confirmationLimitDate, booking),
-            status=booking.status,
-            reimbursedAt=convert_booking_dates_utc_to_venue_timezone(booking.reimbursedAt, booking),
-            isConfirmed=booking.isConfirmed,
-            confirmationDate=convert_booking_dates_utc_to_venue_timezone(booking.confirmationDate, booking),
-            institutionId=booking.institutionId,
-            institutionType=booking.institutionType,
-            institutionName=booking.institutionName,
-            institutionPostalCode=booking.institutionPostalCode,
-            institutionCity=booking.institutionCity,
-            institutionPhoneNumber=booking.institutionPhoneNumber,
-            institutionInstitutionId=booking.institutionInstitutionId,
-            offerName=booking.offerName,
-            offerId=booking.offerId,
-            stockBeginningDatetime=convert_booking_dates_utc_to_venue_timezone(booking.stockBeginningDatetime, booking),
-            venueDepartmentCode=booking.venueDepartmentCode,
-            offererPostalCode=booking.offererPostalCode,
-            numberOfTickets=booking.stockNumberOfTickets,
-        )
+        booking_with_converted_dates = {
+            "collectiveBookingId": booking.collectiveBookingId,
+            "bookedAt": convert_booking_dates_utc_to_venue_timezone(booking.bookedAt, booking),
+            "bookingAmount": booking.bookingAmount,
+            "usedAt": convert_booking_dates_utc_to_venue_timezone(booking.usedAt, booking),
+            "cancelledAt": convert_booking_dates_utc_to_venue_timezone(booking.cancelledAt, booking),
+            "cancellationLimitDate": convert_booking_dates_utc_to_venue_timezone(
+                booking.cancellationLimitDate, booking
+            ),
+            "confirmationLimitDate": convert_booking_dates_utc_to_venue_timezone(
+                booking.confirmationLimitDate, booking
+            ),
+            "status": booking.status,
+            "reimbursedAt": convert_booking_dates_utc_to_venue_timezone(booking.reimbursedAt, booking),
+            "isConfirmed": booking.isConfirmed,
+            "confirmationDate": convert_booking_dates_utc_to_venue_timezone(booking.confirmationDate, booking),
+            "institutionId": booking.institutionId,
```

Comment:
Je ne pense pas que tu as envie de convertir les dates ici. Le rôle d'un repository est purement de récupérer des objets de la base de données, pas de les transformer. Ce genre d'opérations sont plutôt faites dans un serializer

Reply:
Du coup ici tu ne devrais pas avoir de boucle for et renvoyer direct les collective_bookings_page

Reply:
Alors en effet, au préalable la boucle for a été enlevé sauf qu'une page entiere de test pete si je le fais ainsi. Et comme le ticket specifie que c'est un refacto et qu'il ne faut pas changer les tests...